client/components/backup-retention-management/loading/style.scss

```diff
@@ -0,0 +1,16 @@
+.backup-retention-management__loading {
+	&.loading {
```

Comment:
If this adds an extra `loading` class, why do we need the `__loading` suffix for `backup-retention-management__loading` ?